### PR TITLE
docs: Draft of 0.16 release notes

### PR DESCRIPTION
**Pull request by moorereason**
Also included is an enhancement to the gh shortcode to support sending commit
hashes by prepending "0x" to the hash.

### CODE REVIEW

**Comment by bep** [comment]
Looks good. I don't see @spf13 s "partial source reading" feature in there; it was buggy and created some gray hairs, but is important. This needs an ingress that points out the most important new stuff, which would be that and the Go 1.6 block syntax support.

**Comment by moorereason** [comment]
I knew I was probably missing something big.  Is that the incremental build features?  Looking through the `git log`, I couldn't make heads or tails of the logs.  And I think it was merged without a PR, too, so it was hard to track down what happened.

File: docs/content/meta/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,67 @@ title: Release Notes
...
+* 284 (+50) contributors
+* 77 (+10) themes
+
+### New Features
+
+* Add emoji support {{<gh 1892>}}
+* Add `themesDir` option to configuration {{<gh 1556>}}
+* Add support for Go 1.6 `block` keyword in templates {{<gh 1832>}}
+* Partial static sync {{<gh 1644>}}
+* Source file based relative linking (a la Github) {{<gh 0x0f6b334b6715253b030c4e783b88e911b6e53e56>}}
```
Comment (SvenDowideit) [inline docs/content/meta/release-notes.md]: I'm doing some of the cleanup on this atm - and will add some docs.

its worth noting that right now, its a blackfriday only feature.

**Comment by anthonyfok** [comment]
Not sure if it is worth mentioning it in the release notes, but it is a minor change of behaviour now that a trailing backslash in a `.md` file will translate to a `<br />`, see #1935, russross/blackfriday#163 and commit 2c5e4f7

**Comment by bep** [comment]
Merged in f52b040 -- I have some bullet points to add so the draft can live in master.

**Pull request closed without merging**